### PR TITLE
Dw 2687 and dw 2688

### PR DESCRIPTION
**Pull request by John-Milne**
This branch covers both Jira's DW-2687 and DW-2688

### CODE REVIEW

File: repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/heroes/base-hero.ftl [hunk 1]
```diff
@@ -95,11 +99,13 @@ default|backgroundImage|image|accentedImage|accentedImageMirrored
...
         </div>
 
-        <#if heroType == "image" || heroType == "backgroundImage" || heroType == "accentedImage" || heroType == "accentedImageMirrored">
+        <#if heroType = "blackHero" || heroType = "whiteHero" || heroType = "blackBackground" || heroType == "image" || heroType == "backgroundImage" || heroType == "accentedImage" || heroType == "accentedImageMirrored">
             <#assign imageClasses = ""/>
             <#assign style = ""/>
             <#if heroType == "image">
-                <#assign imageClasses = "nhsd-a-image--maintain-ratio nhsd-a-image--position-right"/>
+            	<#assign imageClasses = "nhsd-a-image--maintain-ratio nhsd-a-image--position-right"/>
+            <#elseif heroType = "blackBackground">
```
Comment (GeoffreyHayward) [inline repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/heroes/base-hero.ftl:107]: Should they be double equals here?

**Review by GeoffreyHayward — COMMENTED** [review]
A question to check. Other than that it looks okay.

**Comment by GeoffreyHayward** [comment]
Also, don't forget to rebase down to one commit.

File: repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/heroes/base-hero.ftl [hunk 2]
```diff
@@ -95,11 +99,13 @@ default|backgroundImage|image|accentedImage|accentedImageMirrored
             </div>
         </div>
 
-        <#if heroType == "image" || heroType == "backgroundImage" || heroType == "accentedImage" || heroType == "accentedImageMirrored">
+        <#if heroType = "blackHero" || heroType = "whiteHero" || heroType = "blackBackground" || heroType == "image" || heroType == "backgroundImage" || heroType == "accentedImage" || heroType == "accentedImageMirrored">
```
[the remark this answers is not in the document]
Comment (John-Milne) [reply repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/heroes/base-hero.ftl:102]: Well spotted

[the remark this answers is not in the document]
Comment (John-Milne) [reply repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/heroes/base-hero.ftl]: resolved

**Review by Deven-Rawat — APPROVED** [review]

**Pull request merged**